db_upgrade_scripts/mosip_regprc/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql

```diff
@@ -18,11 +18,60 @@ ALTER TABLE regprc.reg_bio_ref DROP CONSTRAINT IF EXISTS fk_regref_reg CASCADE;
 ALTER TABLE regprc.reg_lost_uin_det DROP CONSTRAINT IF EXISTS fk_rlostd_reg CASCADE;
 ALTER TABLE regprc.registration_transaction DROP CONSTRAINT IF EXISTS fk_regtrn_reg CASCADE;
 
-\ir ../ddl/regprc-additional_info_request.sql
-
-\ir ../ddl/regprc-anonymous_profile.sql
-
-\ir ../ddl/regprc-reg_verification.sql
+CREATE TABLE regprc.additional_info_request(
```

Comment:
Should this be create or alter? So it can be used in future?

Reply:
this is the new table added as part of 1.2.0.1  and not present in 1.1.5.5.